journal-letters/modernized/02-chapter-1.adoc

```diff
@@ -1,5 +1,6 @@
-== Life of William Caton.Chapter I.
+== Life of William Caton. Chapter I
 
+[.chapter-subtitle--blurb]
```

Comment:
Instead of a chapter-subtitle-blurb, this should be a `[.chapter-synopsis]` the double-dashes indicate the separations between lines (items) of the synopsis.